crates/ide-diagnostics/src/handlers/trait_impl_missing_assoc_item.rs

```diff
@@ -13,6 +13,7 @@ pub(crate) fn trait_impl_missing_assoc_item(
 ) -> Diagnostic {
     let missing = d.missing.iter().format_with(", ", |(name, item), f| {
         f(&match *item {
+            hir::AssocItem::Function(func) if func.is_async(ctx.sema.db) => "`async fn ",
```

Comment:
Let's not do that, the important information in the diagnostic here is what item is missing, we aren't reporting the entire signature in the first place to reduce information bloat.

Reply:
Done